### PR TITLE
Fix generated javascript code

### PR DESCRIPTION
**Pull request by mayitbeegh**
Signed-off-by: Carina Ursu <carina@union.ai>

# TL;DR
Use the default pbjs generator to avoid import errors in clients
SyntaxError: Cannot use import statement outside a module

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by ursucarina — APPROVED** [review]

**Review by wild-endeavor — APPROVED** [review]

**Comment by mayitbeegh** [comment]
Replaced by https://github.com/flyteorg/flyteidl/pull/335
